Makefile

```diff
@@ -22,6 +23,12 @@ vendor/ramda.js: node_modules/ramda/dist/ramda.js
 vendor/%.js: node_modules/%/index.js
 	cp '$<' '$@'
 
+js/hm-search.js: node_modules/hindley-milner-search/hm-search.js Makefile
+	$(BROWSERIFY) --standalone HMS -- '$<' >'$@'
```

Comment:
Excellent! I've updated the makefile to simply copy __hms.js__, so we no longer depend on Browserify. :)